src/PluginMediaStreamRenderer.swift

```diff
@@ -27,16 +27,16 @@ class PluginMediaStreamRenderer : NSObject, RTCEAGLVideoViewDelegate {
 		// It's placed over the elementView.
 		self.videoView = RTCEAGLVideoView()
 
-		self.webView.addSubview(self.elementView)
-		self.webView.bringSubviewToFront(self.elementView)
-
 		self.elementView.userInteractionEnabled = false
 		self.elementView.hidden = true
-		self.elementView.backgroundColor = UIColor.blackColor()
+		self.elementView.backgroundColor = UIColor.clearColor()
```

Comment:
Why don't you let the video view to be black when no video is rendered (as a common video element)?